www/docs/en/dev/cordova/events/events.md

```diff
@@ -22,29 +22,561 @@ title: Events
+        <td data-col="amazon-fireos" class="n"></td>
+        <td data-col="android"    class="y"></td>
+        <td data-col="blackberry10" class="y"></td>
+        <td data-col="ios"        class="n"></td>
+        <td data-col="winphone8"  class="n"></td>
+        <td data-col="win"       class="n"></td>
+    </tr>
+
+    <tr>
+        <th><a href="#link-volumeupbutton">volumeupbutton</a></th>
+        <td data-col="amazon-fireos" class="n"></td>
+        <td data-col="android"    class="y"></td>
+        <td data-col="blackberry10" class="y"></td>
+        <td data-col="ios"        class="n"></td>
+        <td data-col="winphone8"  class="n"></td>
+        <td data-col="win"       class="n"></td>
+    </tr>    
+</tbody>
+</table>
+
+<!-- END HTML -->
+
+
+**Note**: Applications typically should use `document.addEventListener` to
+attach an event listener once the [deviceready](#link-deviceready) 
+event fires.
+
+## deviceready
+
+The deviceready event fires when Cordova is fully loaded. This event is 
+essential to any application. It signals that Cordova's device APIs have 
+loaded and are ready to access.
+
+Cordova consists of two code bases: native and JavaScript. While the
+native code loads, a custom loading image displays. However,
+JavaScript only loads once the DOM loads. This means the web app may
+potentially call a Cordova JavaScript function before the
+corresponding native code becomes available.
+
+The `deviceready` event fires once Cordova has fully loaded. Once the
+event fires, you can safely make calls to Cordova APIs.  Applications
+typically attach an event listener with `document.addEventListener`
+once the HTML document's DOM has loaded.
+
+The `deviceready` event behaves somewhat differently from others.  Any
+event handler registered after the `deviceready` event fires has its
+callback function called immediately.
+
+### Quick Example
+
+    document.addEventListener("deviceready", onDeviceReady, false);
+
+    function onDeviceReady() {
+        // Now safe to use device APIs
+    }
+
+### Full Example
+
+**Note**: For the sake of readability, all the other events assume that you use the
+below html code.
+
+#### HTML File
+
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <title>Device Ready Example</title>
+
+        <script type="text/javascript" charset="utf-8" src="cordova.js"></script>
+        <script type="text/javascript" charset="utf-8" src="example.js"></script>
+      </head>
+      <body onload="onLoad()">
+      </body>
+    </html>
+
+#### JS File
+    // example.js file
+    // Wait for device API libraries to load
+    //
+    function onLoad() {
+        document.addEventListener("deviceready", onDeviceReady, false);
+    }
+
+    // device APIs are available
+    //
+    function onDeviceReady() {
+        // Now safe to use device APIs
+    }
+
+## pause
+
+The pause event fires when an application is put into the background. The `pause` 
```

Comment:
That was intentional. To explicitly say, this is just a JS file and also what the following function is used.